packages/core-data/src/reducer.js

```diff
@@ -238,8 +238,8 @@ function entity( entityConfig ) {
 		// Inject the entity config into the action.
 		replaceAction( ( action ) => {
 			return {
-				...action,
 				key: entityConfig.key || DEFAULT_ENTITY_KEY,
+				...action,
```

Comment:
Just to make sure I'm following: this is to ensure that the action can override the injected entity config key?

Reply:
That's the plan! 😄